### PR TITLE
AC-764 Updated Contact Screen UI

### PR DESCRIPTION
**Pull request by rishabh-997**
## Description of what I changed
1. Changed UI
2. Added a necessary font- RobotoSemiBold
3. Removed unused previous icons 

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-764

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
Screenshot_2020-05-20-17-43-33-41_0b570628ec88a4202b7916ec2df1bebf

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -376,15 +376,16 @@
 
 	<!-- Contact Activity -->
 	<string name="helpdesk_mail" translatable="false">helpdesk@openmrs.com</string>
-	<string name="go_to_forum">Go To Forum</string>
-	<string name="go_to_irc">Go To IRC</string>
-	<string name="open">Open</string>
+	<string name="go_to_forum">Forum</string>
+	<string name="go_to_irc">IRC</string>
+	<string name="open">Go To</string>
```
Comment (VibhorChinda) [inline openmrs-client/src/main/res/values/strings.xml:381]: I feel it should be renamed to "go_to" not "open"

File: openmrs-client/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -376,15 +376,16 @@
 
 	<!-- Contact Activity -->
 	<string name="helpdesk_mail" translatable="false">helpdesk@openmrs.com</string>
-	<string name="go_to_forum">Go To Forum</string>
-	<string name="go_to_irc">Go To IRC</string>
-	<string name="open">Open</string>
+	<string name="go_to_forum">Forum</string>
```
Comment (VibhorChinda) [inline openmrs-client/src/main/res/values/strings.xml:379]: change "go_to_forum" to "forum"

same with irc

**Comment by VibhorChinda** [comment]
@rishabh-997 please attach the screenshot for hind translation screen as well. I feel in hindi the text meaning will something different then needed.

Also i feel the text message at the bottom is quite long. How about changing it. Like something simple like "We will be happy to help you".

What are your views @f4ww4z sir on it.

**Comment by f4ww4z** [comment]
> Also i feel the text message at the bottom is quite long. How about changing it. Like something simple like "We will be happy to help you".

@VibhorChinda  I agree with this, the more concise the better. Change to "Our team will try our best to help you. Please reach out to us by using the services above."

1. It would be better if the orange buttons had a drop shadow just like in the design
2. Change `Go To` to `Go to` (lowercase `to`)

**Comment by rishabh-997** [comment]
done with the changes....
Screenshot_2020-05-21-23-04-29-78_0b570628ec88a4202b7916ec2df1bebf

**Comment by VibhorChinda** [comment]
LGTM 👍

**Comment by f4ww4z** [comment]
@rishabh-997 Just need to fix the merge conflict caused by the code formatting of #723

**Pull request merged**